js/src/api/rpc/parity/parity.js

```diff
@@ -60,6 +60,12 @@ export default class Parity {
       .then(outNumber);
   }
 
+  dappsInterface () {
+    return this._transport
+      .execute('parity_dappsInterface')
+      .then(outNumber);
```

Comment:
It returns a string/IP, so would remove this line. (Just the execute is fine without a formatter)